src/elm/Page/Community/Settings/Features.elm

```diff
@@ -0,0 +1,259 @@
+import Html.Attributes exposing (checked, class, for, id, name, style, type_)
+import Html.Events exposing (onCheck)
+import I18Next exposing (Translations, t)
+import Json.Encode
+import Page
+import Ports
+import Route
+import Session.LoggedIn as LoggedIn exposing (External(..), FeedbackStatus(..))
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> Symbol -> ( Model, Cmd Msg )
+init { shared } symbol =
+    ( initModel symbol
+    , Api.Graphql.query shared (Community.communityQuery symbol) CompletedLoad
+    )
+
+
+initModel : Symbol -> Model
+initModel symbol =
+    { status = Loading
+    , symbol = symbol
+    , hasShop = False
+    , hasObjectives = False
+    }
+
+
+type alias Model =
+    { status : Status
+    , symbol : Symbol
+    , hasShop : Bool
+    , hasObjectives : Bool
+    }
+
+
+type Status
+    = Loading
+    | LoadingFailed (Graphql.Http.Error (Maybe Community.Model))
+    | Loaded Community.Model
+
+
+type Feature
+    = Shop
+    | Objectives
+
+
+type Msg
+    = CompletedLoad (Result (Graphql.Http.Error (Maybe Community.Model)) (Maybe Community.Model))
+    | ToggleShop Bool
+    | ToggleActions Bool
+    | Ignored
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+view : LoggedIn.Model -> Model -> { title : String, content : Html Msg }
+view loggedIn model =
+    let
+        title =
+            "Features"
+
+        content =
+            let
+                translations =
+                    loggedIn.shared.translations
+
+                translate =
+                    t translations
+            in
+            div [ class "bg-white flex flex-col items-center" ]
+                [ Page.viewHeader loggedIn "Features" (Route.CommunitySettings model.symbol)
+                , div
+                    [ class "container w-full divide-y"
+                    ]
+                    [ toggleView translations (translate "community.objectives.title_plural") model.hasObjectives ToggleActions "actions"
+                    , toggleView translations (translate "menu.shop") model.hasShop ToggleShop "shop"
+                    ]
+                ]
+    in
+    { title = title
+    , content = content
+    }
+
+
+toggleView : Translations -> String -> Bool -> (Bool -> Msg) -> String -> Html Msg
+toggleView translations labelText isEnabled toggleFunction inputId =
+    let
+        translate =
+            t translations
+
+        classes =
+            class "flex items-center"
+
+        statusText =
+            if isEnabled then
+                translate "settings.features.enabled"
+
+            else
+                translate "settings.features.disabled"
+    in
+    div
+        [ class "grid w-full py-4"
+        , style "grid-template" """
+                                'label status toggle' 40px / auto 80px 50px
+                                """
+        ]
+        [ span [ classes, style "grid-area" "label" ] [ text labelText ]
```

Comment:
The text seems bigger than the rest of the application. Maybe the whole app is wrong, or only this text. Either way we need to keep our deliveries consistent